### PR TITLE
enable apps/v1beta2 for pull-kubernetes-e2e-gce-etcd3

### PR DESCRIPTION
**Pull request by crimsonfaith91**
Addressing #3841

### CODE REVIEW

**Comment by crimsonfaith91** [comment]
/assign @krzyzacy 
Do you mind to review it? Thanks!

File: jobs/pull-kubernetes-e2e-gce-etcd3.env [hunk 1]
```diff
@@ -4,3 +4,6 @@ KUBE_NODE_OS_DISTRIBUTION=debian
 # Panic if anything mutates a shared informer cache
 ENABLE_CACHE_MUTATION_DETECTOR=true
 
+# Enable apps/v1beta2
+# TODO (juntee): remove this when k8s v1.8 is out 
+RUNTIME_CONFIG=apps/v1beta2
```
Comment (krzyzacy) [inline jobs/pull-kubernetes-e2e-gce-etcd3.env]: new line :-)

Comment (crimsonfaith91) [inline jobs/pull-kubernetes-e2e-gce-etcd3.env]: do you mean adding a new line after `RUNTIME_CONFIG=apps/v1beta2`?

Comment (krzyzacy) [inline jobs/pull-kubernetes-e2e-gce-etcd3.env]: yeah, each file should end with a new line

Comment (crimsonfaith91) [inline jobs/pull-kubernetes-e2e-gce-etcd3.env]: i added the new line, but it is not appearing here.

**Comment by krzyzacy** [comment]
/lgtm

I see the current etcd3 presubmit is still pretty green though

File: jobs/pull-kubernetes-e2e-gce-etcd3.env (hunk 1 above)
Comment (krzyzacy) [inline jobs/pull-kubernetes-e2e-gce-etcd3.env]: it had a 'need new line' icon if you don't add one

**Comment by crimsonfaith91** [comment]
i added the new line explicitly by having 2 new lines, so the new line appears now...

**Comment by krzyzacy** [comment]
one line in the end is correct - if you look at other files.

**Comment by crimsonfaith91** [comment]
no, i meant 2 lines on my editor will translate into 1 line on the changed files. perhaps it is due to the editor setting, but the env file now only has 1 blank line at the end.

**Comment by krzyzacy** [comment]
:smiling_imp:

**Comment by crimsonfaith91** [comment]
@krzyzacy Thanks a lot for the help! I should have consulted you rather than spending several hours changing my codes. Have a nice day! :)

**Comment by crimsonfaith91** [comment]
@krzyzacy @jpbetz The skipping above is due to no resources. Based on `runtime-config` flag description found [here](https://kubernetes.io/docs/admin/kube-apiserver/), perhaps we need to have something like `RUNTIME_CONFIG=apps/v1beta2/statefulset` to enable the resources.

**Comment by foxish** [comment]
Runtime config should be: `"apps/v1beta2,apps/v1beta2/statefulsets=true"` for this to work.

**Comment by foxish** [comment]
We should think further about how far we need this to go. Although the e2es will pass in the submit-queue with this change, they'll likely remain broken in the other CI clusters if they **also** don't enable said group. Maybe the right thing to do is to enable `apps/v1beta2` via runtime config in all the CI clusters? 

cc @kow3ns

**Comment by crimsonfaith91** [comment]
@foxish thanks!

**Pull request merged**